ripe/atlas/tools/commands/alias.py

```diff
@@ -0,0 +1,170 @@
+# Copyright (c) 2016 RIPE NCC
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+
+from ..exceptions import RipeAtlasToolsException
+from ..helpers.validators import ArgumentType
+from ..settings import Aliases, aliases
+from .base import Command as BaseCommand
+
+
+class Command(BaseCommand):
+
+    NAME = "alias"
+
+    EDITOR = os.environ.get("EDITOR", "/usr/bin/vim")
+    DESCRIPTION = "Manage measurements' and probes' aliases"
+    EXTRA_DESCRIPTION = (
+        "As an alternative to this command, you can just create/edit {}"
+        .format(Aliases.USER_RC)
+    )
+
+    def add_arguments(self):
+        subparsers = self.parser.add_subparsers(
+            title="action",
+            dest="action",
+            help="Action to be performed on aliases. "
+                 "Run 'ripe-atlas alias <action> --help' for more details."
+        )
+
+        parser = subparsers.add_parser(
+            "add",
+            help="Add/modify an alias."
+        )
+        parser.add_argument(
+            "type",
+            action="store",
+            choices=["measurement", "probe"],
+            help="Type of target object.",
+        )
+        parser.add_argument(
+            "target",
+            action="store",
+            type=int,
+            help="Target's ID.",
+        )
+        parser.add_argument(
+            "alias",
+            action="store",
+            type=ArgumentType.alias_is_valid,
+            help="Alias name.",
+        )
+
+        parser = subparsers.add_parser(
```

Comment:
For the shake of readability I wouldn't reuse the same parser name for all subparsers. I would do add_parser, del_parser, show_parser